docs/management/admin/endpoint-command-ref.asciidoc

```diff
@@ -0,0 +1,343 @@
+[discrete]
+=== Example
+
+[source,shell]
+------
+sudo /Library/Elastic/Endpoint/elastic-endpoint install --upgrade --resources endpoint-security-resources.zip
+------
+
+[discrete]
+[[elastic-endpoint-memorydump-command]]
+== elastic-endpoint memorydump
+
+Save a memory dump of the {elastic-endpoint} service.
+
+[discrete]
+=== Options
+
+`--compress`::
+Compress the saved memory dump.
+
+`--timeout <duration>`::
+Specify the memory collection timeout, in seconds; the default is 60 seconds.
+
+[discrete]
+=== Example
+
+[source,shell]
+------
+sudo /Library/Elastic/Endpoint/elastic-endpoint memorydump --timeout 120
+------
+
+[discrete]
+[[elastic-endpoint-run-command]]
+== elastic-endpoint run
+
+Run `elastic-endpoint` as a foreground process if no other instance is already running.
+
+[discrete]
+=== Example
+
+[source,shell]
+------
+sudo /Library/Elastic/Endpoint/elastic-endpoint run
+------
+
+[discrete]
+[[elastic-endpoint-send-command]]
+== elastic-endpoint send
+
+Send the requested document to the {stack}.
+
+[discrete]
+=== Subcommands
+
+`metadata`::
+Send an off-schedule metrics document to the {stack}.
+
+[discrete]
+=== Example
+
+[source,shell]
+------
+sudo /Library/Elastic/Endpoint/elastic-endpoint send metadata
+------
+
+[discrete]
+[[elastic-endpoint-status-command]]
+== elastic-endpoint status
+
+Retrieve the current status of the running {elastic-endpoint} service. The command also returns the last known status of {agent}.
+
+[discrete]
+=== Options
+
+`--output`::
+Control the level of detail and formatting of the information. Valid values are:
+
+* `human`: Returns limited information when {elastic-endpoint}'s status is `Healthy`. If any policy actions weren't successfully applied, the relevant details are displayed.
+* `full`: Always returns the full status information.
+* `json`: Always returns the full status information.
+
+[discrete]
+=== Example
+
+[source,shell]
+------
+sudo /Library/Elastic/Endpoint/elastic-endpoint status --output json
+------
+
+[discrete]
+[[elastic-endpoint-test-command]]
+== elastic-endpoint test
+
+Perform the requested test.
+
+[discrete]
+=== Subcommands
+
+`output`::
+Test whether {elastic-endpoint} can connect to remote resources.
+
+[discrete]
+=== Example
+
+[source,shell]
+------
+sudo /Library/Elastic/Endpoint/elastic-endpoint test output
+------
+
+[discrete]
+=== Example output
+
+[source,txt]
+----
+Testing output connections
+
+Using proxy:
+
+Elasticsearch server: https://example.elastic.co:443
+        Status: Success
+
+Global artifact server: https://artifacts.security.elastic.co
+        Status: Success
+
+Fleet server: https://fleet.example.elastic.co:443
+        Status: Success
+----
+
+[discrete]
+[[elastic-endpoint-top-command]]
+== elastic-endpoint top
+
+Show a breakdown of the executables that triggered {elastic-endpoint} CPU usage within the last interval. This utility displays which {elastic-endpoint} features are resource-intensive for a particular executable.
```

Comment:
Another word choice nit: this is just a single command, but typically I think of a "utility" more as a small application that performs several different actions or commands.
```suggestion
Show a breakdown of the executables that triggered {elastic-endpoint} CPU usage within the last interval. This displays which {elastic-endpoint} features are resource-intensive for a particular executable.
```